lib/buildkite/test_collector.rb

```diff
@@ -101,5 +101,11 @@ def self.enable_tracing!
         Buildkite::TestCollector::Uploader.tracer&.backfill(:sql, finish - start, **{ query: payload[:sql] })
       end
     end
+
+    def self.safe(&block)
+      block.call
+    rescue StandardError => e
+      logger.error("Buildkite::TestCollector received exception: #{e}")
```

Comment:
Does this also log the backtrace? If not, would it help with debugging the error?

Reply:
Good point. I'll add it in.